packages/nuxt/package.json

```diff
@@ -24,11 +24,7 @@
   ],
   "scripts": {
     "build": "nuxt-module-build",
-    "prepack": "nuxt-module-build",
-    "dev": "nuxi dev playground",
-    "dev:build": "nuxi build playground",
-    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
-    "dependenciesInstalled": "node ./scripts/stud-nuxt.mjs"
+    "prepack": "nuxt-module-build"
```

Comment:
Aren't they equal?

Reply:
I mean `build` and `prepack` are running the same command.

Reply:
prepack is used by npm and runs before npm publish. Build is used by us to get thing builded.